<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>401</width>
      <height>291</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>500</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>退出系统</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>450</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>色块阈值</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openbt">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>打开串口</string>
    </property>
   </widget>
   <widget class="QLabel" name="qrCodeLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>131</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>端口</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>波特率</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox_2">
       <property name="currentText">
        <string>1200</string>
       </property>
       <item>
        <property name="text">
         <string>1200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4800</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>19200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>38400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>57600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>115200</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>数据位</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBox_3">
       <property name="currentText">
        <string>5</string>
       </property>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>停止位</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboBox_4">
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>校验位</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="comboBox_5">
       <item>
        <property name="text">
         <string>none</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>奇校验</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>偶校验</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="sendEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="recvEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>221</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>440</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>清空接收</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_send_hex">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>82</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>HEX发送</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_send_line">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>82</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>发送新行</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>290</y>
      <width>401</width>
      <height>261</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>序号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>出发地</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>目的地</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>物品类型</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>390</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>清空表格</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>82</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>二维码</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>82</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>色块</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>320</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>打开传送带</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
